Sources/TuistCore/Models/InfoPlist.swift

```diff
@@ -44,14 +44,15 @@ public enum InfoPlist: Equatable {
     }
 
     case file(path: AbsolutePath)
+    case generatedFile(path: AbsolutePath)
```

Comment:
Note: this was needed to prevent the `TargetLinter` from flagging this path as one that doesn't exist.

e.g. when running generate for the first time.

Reply:
@kwridan what about adding an explanatory line in case someone comes across this case in the future and wonder what's the difference?